### ISSUE TITLE
Implement Gtk Application support

### ISSUE THREAD

**Issue by dahenson** [conformal/gotk3#48]
https://developer.gnome.org/gtk3/stable/Application.html

I want this for a personal project of mine, and I already have some of the code working locally. I will attempt a pull request when I feel comfortable with it.

The biggest change will be the addition of some [functions from gio](https://developer.gnome.org/gio/stable/application.html). Feel free to stop me if you think this work will be unnecessary.

**Comment by jrick** [comment]
Yes, I was going to do it but got sidetracked with other projects.  Thanks for taking the time to look into this.
